.dockerignore

```diff
@@ -0,0 +1,126 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+*.DS_Store
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+scylla/assets/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
```

Comment:
We're not using `PyInstaller`. Could you remove it?

Reply:
Since `.dockerignore` is just a copy of `.gitignore ` I do not review it🤣 

So, maybe we could remove it from both files?

Reply:
Not a blocker for this PR. We can clean it up later.